### ISSUE TITLE
No instance for HasLink when using a Header

### ISSUE THREAD

**Issue by kantp** [haskell-servant/servant#128]
Hi again,

while trying to generate docs (via docsWith) for an API using Request Headers, I got the error

```
No instance for (HasLink
                   (Header "my-header" Text :> Get '[JSON, PlainText] [Text]))
  arising from a use of ‘extraInfo’
```

My guess is that there is an instance declaration missing in Link.hs. Or am I doing something wrong here?

Best,
Philipp

**Comment by alpmestan** [comment]
Cooked up a little patch for this, just waiting for travis. Thank you very much for reporting this! We'll probably include this in the bugfix release we'll make soon.

**Comment by kantp** [comment]
Thanks, that was fast!

**Comment by mf59816** [comment]
cool, release!  (-:

will that also include #111?  we're are all very excited about that over here at https://github.com/liqd/thentos.

(sorry, that was very off-topic.)

**Comment by alpmestan** [comment]
@mf59816 The bugfix release won't include #111 unfortunately, because it contains breaking changes and will have to be released as **0.5**. I don't have any timeline in mind for 0.5 though -- what are you thoughts on this @jkarni ? We were talking about having the auth stuffs in 0.5 out of the box too, which means it wouldn't be out before several weeks or more.
